ghost/i18n/locales/pt-BR/newsletter.json

```diff
@@ -5,10 +5,10 @@
     "Email": "E-mail",
     "free": "grátis",
     "Keep reading": "Continue lendo",
-    "Less like this": "Não gostei",
+    "Less like this": "Não gostei, não mostrar mais como este",
```

Comment:
This is a label for the thumbs up or down icon in the newsletter.  It needs to be short.

Reply:
Pull request updated. Thank you very much for the input.